test-exports-cjs/import.js

```diff
@@ -37,6 +38,13 @@ async function test() {
   );
 
   assert((await vs.similaritySearchVectorWithScore([0, 0, 1], 1)).length === 1);
+
+  // Test CSVLoader
+  const loader = new CSVLoader(new Blob(["a,b,c\n1,2,3\n4,5,6"]));
+
+  const docs = await loader.load();
```

Comment:
`loader.load` feels a lil off; I understand why it's separate from the ctor cause it's async. maybe `loader.init`?

not sure..

Reply:
Hmm yea I think we went with this because it's the name used in python